.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: test-e2e
```

Comment:
[optional] can the file name please match the workflow name so it's easy to jump from one to the other? 

Concretely the file name should be test-e2e.yml if that's the name used here.

Reply:
done